### DISCUSSION TITLE
User Management Names

### DISCUSSION THREAD

**Discussion by rcpacheco** [oqtane/oqtane.framework#3225] [Q&A]
In the User Management-Profile there is a  FirstName and a LastName field. In the User Management-Profile there is a field FullName.
The FullName is not derived from the FirstName and LastName from profile. Is it realy necessay? Shouldn't it be calculated or eliminated from the Identity Tab?

**Comment by sbwalker** [comment]
@rcpacheco the Full Name in the User table is essentially a "friendly name" for the user (as usernames are often not friendly). It could be a user's full name, a nickname, or whatever - there is no validation which enforces a convention. By default the Profile for each site also has individual fields for First Name and Last Name - however you are free to modify the Profile fields however you see fit - you could remove them all, add additional items, etc...
